src/Data/Connection/AbstractConnectionResolver.php

```diff
@@ -409,7 +409,13 @@ public function get_amount_requested() {
 			}
 		}
 
-		return max( 0, $amount_requested );
+        /**
```

Comment:
looks like spacing is a bit funky between the comment and the code?

Reply:
Converted the spaces to tabs. If that's what you mean?

Reply:
ah there's code sniffer workflow